app/serializers/search/data_file_serializer.rb

```diff
@@ -0,0 +1,27 @@
+class Search::DataFileSerializer < ActiveModel::Serializer
+  attributes :id,
+             :name,
+             :is_deleted,
+             :created_at,
+             :updated_at,
+             :label,
+             :meta
+
+  has_many :tags, serializer: Search::TagSummarySerializer
+  has_one :project, serializer: Search::ProjectSummarySerializer
+  has_one :parent, serializer: Search::FolderSummarySerializer
+  has_one :creator, serializer: Search::UserSummarySerializer
+  
+  def is_deleted
+    object.is_deleted?
+  end
+
+  def meta
```

Comment:
I dont know? It certainly doesnt lend itself to it. It wants a meta hash, with a series of key (meta_template.template.name): Hash with series of key (meta_property.property.key): meta_property.value entries. These do not align nicely with the underlying object structures